Blish HUD/GameServices/Gw2Mumble/PlayerCamera.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using Gw2Sharp.Mumble;
+using Microsoft.Xna.Framework;
+
+namespace Blish_HUD.Gw2Mumble {
+    public class PlayerCamera {
+
+        private readonly Gw2MumbleService _service;
+
+        private Vector3 _position    = Vector3.Zero;
+        private Vector3 _forward     = Vector3.Forward;
+        private float   _fieldOfView = 1;
+
+        private Matrix _view;
+        private Matrix _playerView;
+        private Matrix _projection;
+        private Matrix _worldViewProjection;
+
+        /// <summary>
+        /// The closest distance entities are rendered.
+        /// </summary>
+        public float NearPlaneRenderDistance { get; set; } = 0.01f;
+
+        /// <summary>
+        /// The farthest distance entities are rendered.
+        /// </summary>
+        public float FarPlaneRenderDistance  { get; set; } = 1000.0f;
+
+        #region Mumble Proxied Fields
+
+        /// <inheritdoc cref="IGw2MumbleClient.CameraPosition"/>
+        public Vector3 Position => _position;
+
+        /// <inheritdoc cref="IGw2MumbleClient.CameraFront"/>
+        public Vector3 Forward => _forward;
+
+        /// <inheritdoc cref="IGw2MumbleClient.FieldOfView"/>
+        public float FieldOfView => MathHelper.Clamp(_fieldOfView, 0.01f, (float)Math.PI - 0.01f);
```

Comment:
Put the clamp into the Update function, so it doesn't need to be calculated whenever someone reads it. We need it at least once anyway, so we can only gain here.